lib/run.js

```diff
@@ -115,7 +121,15 @@ exports.builder = yargs => {
       if (argv.script) {
         argv.script = argv.script.map(script => resolve(script))
       }
-
+      if (argv.preload) {
+        try {
+          argv.preload = join(process.cwd(), argv.preload)
```

Comment:
Is would suggest to use `resolve` here, which should implicitly use the current working directory for relative paths, and will not lead to surprises if you configure an absolute path.

Reply:
Yep, totally agree 👍🏻 

Should be fine now 😄